pkg/packagekit/updates.jsx

```diff
@@ -23,6 +23,9 @@ var moment = require("moment");
 var Tooltip = require("cockpit-components-tooltip.jsx").Tooltip;
 require("listing.less");
 
+// for interactively testing layout changes with large updates
+//import { injectMockUpdates } from "./mock-updates.es6";
```

Comment:
Agreed. I changed the commit to instead just explain how to use the mock updates in a comment in mock-updates.es6, and dropped these comments from `updates.jsx`. It's still easy enough that way. 

Our integration tests do something similar, but it's annoying to have to wait for an entire  image rebuild and test run to test a little CSS change that way. So I'd like to keep that additional file for the time being.